test/xUnit/csharp/test_CommandLineParser.cs

```diff
@@ -103,18 +103,27 @@ public static void TestDefaultParameterIsFileName_Not_Exist(params string[] comm
         [Fact]
         public static void TestDefaultParameterIsFileName_Exist()
         {
-            var fileName = System.IO.Path.GetTempFileName();
+            var tempFile = System.IO.Path.GetTempFileName();
+            var tempPs1 = tempFile + ".ps1";
+            File.Move(tempFile, tempPs1);
 
             var cpp = new CommandLineParameterParser();
 
-            cpp.Parse(new string[] { fileName });
+            cpp.Parse(new string[] { tempPs1 });
 
-            Assert.False(cpp.AbortStartup);
-            Assert.False(cpp.NoExit);
-            Assert.False(cpp.ShowShortHelp);
-            Assert.False(cpp.ShowBanner);
-            Assert.Equal(CommandLineParameterParser.NormalizeFilePath(fileName), cpp.File);
-            Assert.Null(cpp.ErrorMessage);
+            try
+            {
+                Assert.False(cpp.AbortStartup);
+                Assert.False(cpp.NoExit);
+                Assert.False(cpp.ShowShortHelp);
+                Assert.False(cpp.ShowBanner);
+                Assert.Equal(CommandLineParameterParser.NormalizeFilePath(tempPs1), cpp.File);
+                Assert.Null(cpp.ErrorMessage);
+            }
+            finally
+            {
+                File.Delete(tempPs1);
```

Comment:
Should we remove tempFile too?

Reply:
tempFile is renamed to tempPs1 so only tempPs1 actually exists